public_html/wp-content/themes/pattern-directory/src/components/pattern-thumbnail/canvas.js

```diff
@@ -55,28 +54,19 @@ function PatternThumbnail( { className, html } ) {
 
 	return (
 		<div
-			className={ className }
 			ref={ wrapperRef }
 			style={ {
 				height: frameHeight,
+				overflow: 'hidden',
 			} }
-			tabIndex="-1"
 		>
-			<Iframe
-				className="pattern-grid__preview-iframe"
+			<iframe
```

Comment:
I think we should aim at using an [image snapshot](https://github.com/WordPress/wporg-mu-plugins/tree/trunk/mu-plugins/blocks/screenshot-preview) in a future PR.

Reply:
Good idea for these thumbnail views  👍🏻